features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/resource-access-control-v2.xml.j2

```diff
@@ -147,6 +147,34 @@
             <Scopes>internal_notification_senders_delete</Scopes>
         </Resource>
 
+        <!-- [Organization]  Certificate Validation Management API -->
+        <Resource context="(.*)/o/api/server/v1/certificate-validation/(.*)" secured="true" http-method="POST">
```

Comment:
Let's update xml files also for consistency

Reply:
fixed with https://github.com/wso2/carbon-identity-framework/pull/6448/commits/f5ee93c59a2b06cf45fd9151d5b3deab0bdf59c1